README.md

```diff
@@ -122,7 +122,7 @@ config.notifiers << {
 }
 ```
 
-6. Add the code below to the environment config file where you are using ExceptionHunter with the correct server url.
+5. Add the code below to the environment config file where you are using ExceptionHunter with the correct server url.
```

Comment:
doesn't
1.
1.
1.
1.

work in this case?

Reply:
Yeah that's a good solution